SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKCloudDeploymentConfiguration.cpp

```diff
@@ -320,15 +322,16 @@ void SSpatialGDKCloudDeploymentConfiguration::Construct(const FArguments& InArgs
 								.FillWidth(1.0f)
 								[
 									SNew(STextBlock)
-									.Text(FText::FromString(FString(TEXT(""))))
-								.ToolTipText(FText::FromString(FString(TEXT(""))))
+									.Text(FText::FromString(FString(TEXT("Automatically Generate Configuration"))))
+								.ToolTipText(FText::FromString(FString(TEXT("Whether to automatically generate the configuration from the current map every time a cloud deployment is started."))))
 								]
-							+ SHorizontalBox::Slot()
+
+								+ SHorizontalBox::Slot()
```

Comment:
The tabs seem off on this, but it looks like the existing code already had problems with this. Super minor, but would be awesome if you could make it consistent.